From c31dca74e68f6d1550692a2e28780499fa4e7dd5 Mon Sep 17 00:00:00 2001
From: Alex Deymo <deymo@google.com>
Date: Fri, 1 Oct 2021 16:27:01 +0200
Subject: [PATCH] Remove sys/platform/ppc.h include. (#678)

According to the comment this include was used for
`__ppc_get_timebase_freq` at some point in this file but nowadays that
function is not there, so we can remove the include. Fixes #677.
---
 tools/cpu/cpu.cc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tools/cpu/cpu.cc b/tools/cpu/cpu.cc
index 24cc097..b75af03 100644
--- a/tools/cpu/cpu.cc
+++ b/tools/cpu/cpu.cc
@@ -18,10 +18,6 @@
 #endif
 #endif
 
-#if JXL_ARCH_PPC
-#include <sys/platform/ppc.h>  // __ppc_get_timebase_freq
-#endif
-
 #if JXL_ARCH_ARM
 #include <unistd.h>  // sysconf
 #endif
-- 
2.33.0

